packages/desktop/webpack.config.js

```diff
@@ -168,7 +168,7 @@ module.exports = [
         },
         mode,
         plugins: [...rendererPlugins, ...sentryPlugins],
-        devtool: prod ? false : 'cheap-module-source-map',
+        devtool: SENTRY && prod ? 'source-map' : 'cheap-module-source-map',
```

Comment:
We want detailed sourcemaps when we submit to Sentry, but we don't really need extensive detail when we're doing local development